source/_integrations/seventeentrack.markdown

```diff
@@ -72,3 +72,14 @@ content: >-
 ```
 
 {% endraw %}
+
+Attributes that can be using in Lovelace markdown:
+| Service data attribute |  Description |
+| ---------------------- |  ----------- |
+| `friendly_name` | Description manually added into 17Track |
```

Comment:
`friendly_name` is not part of the 17Track data, but part of the Home Assistant entity logic itself. As this is not 17Track specific, I think we should not document this here.